edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent/DiagnosticConfig.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+    using Microsoft.Extensions.Configuration;
+
+    public class DiagnosticConfig
+    {
+        public bool Enabled;
+        public string MetricsStoragePath;
+        public TimeSpan ScrapeInterval;
+        public TimeSpan UploadInterval;
+
+        public DiagnosticConfig(bool enabled, string storagePath, IConfiguration configuration)
+        {
+            this.Enabled = enabled;
+            this.MetricsStoragePath = Path.Combine("metrics", storagePath);
+            this.ScrapeInterval = configuration.GetValue<TimeSpan>("metric_scrape_interval");
```

Comment:
why the config key "metric_scrape_interval" is all lowercase and separated words by underscore? this is not consistent with all parameter in appsettings_agent.json.